halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentConfiguration.java

```diff
@@ -87,4 +92,9 @@ public NodeIterator getChildren() {
   public void accept(ConfigProblemSetBuilder psBuilder, Validator v) {
     v.validate(psBuilder, this);
   }
+
+  protected List<String> versionOptions(ConfigProblemSetBuilder psBuilder) {
+    VersionsService service = psBuilder.getContext().getBean(VersionsService.class);
+    return service.getVersions().getVersions().stream().map(Version::getVersion).collect(Collectors.toList());
```

Comment:
`getVersions().getVersions()` is a little tough to read. Could the first be `loadVersions`?